### DISCUSSION TITLE
Incorrect strings extracted from Qt linguist project

### DISCUSSION THREAD

**Discussion by BLumia** [WeblateOrg/weblate#11266] [Q&A] [answered]
Hi,

I'm facing a weird issue recently while using Codeberg's Weblate instance. In [one of my newly created project](https://translate.codeberg.org/projects/pineapple-midi-player/application/en/), all the strings form that newly imported project have a `MainWindow` prefix, but the original `ts` file doesn't have such issue. [My other project on weblate](https://translate.codeberg.org/browse/pineapple-tracker-player/application/en/) with nearly the same configuration doesn't have such issue. I compared those two projects' settings but didn't find anything wrong, maybe I missed something but I'm not quite sure.

I asked this issue in the Codeberg's Matrix chatroom, and was suggested to create a discussion here. Any idea about the reason?

Feel free to let me know If any additional information is needed. Thanks in advance!

**Comment by nijel** [comment, marked as answer]
You have configured it as monolingual while the files are actually bilingual. See https://docs.weblate.org/en/latest/formats.html#bilingual-and-monolingual-formats

**Reply by BLumia** [reply]
Wow, I removed the monolingual file field and now it works! I must made some mistakes while the weblate project creation process.

Thanks!
